addon/components/ilios-calendar-week.js

```diff
@@ -20,17 +20,32 @@ export default Component.extend({
       this.$(".el-calendar .week").scrollTop(500);
     });
   },
-  singleDayEvents: computed('calendarEvents.[]', function(){
-    const events = this.get('calendarEvents');
+  ilmPreWorkEvents: computed('calendarEvents.[]', function () {
+    const calendarEvents = this.calendarEvents || [];
+    const preWork =  calendarEvents.reduce((arr, eventObject) => {
+      return arr.pushObjects(eventObject.prerequisites);
+    }, []);
+
+    return preWork.filter(ev => ev.ilmSession);
+  }),
+
+  nonIlmPreWorkEvents: computed('calendarEvents.[]', function () {
+    const calendarEvents = this.calendarEvents || [];
+    return calendarEvents.filter(ev => {
+      return ev.postrequisites.length === 0 && !ev.ilmSession;
+    });
+  }),
+  singleDayEvents: computed('nonIlmPreWorkEvents.[]', function(){
+    const events = this.get('nonIlmPreWorkEvents');
     if(isEmpty(events)){
       return [];
     }
     return events.filter(
       event => moment(event.startDate).isSame(moment(event.endDate), 'day')
     );
   }),
-  multiDayEventsList: computed('calendarEvents.[]', function(){
-    const events = this.get('calendarEvents');
+  multiDayEventsList: computed('nonIlmPreWorkEvents.[]', function(){
```

Comment:
i see you shortened this to `multiDayEvents` elsewhere. is this an omission here, or legit?

Reply:
nvm